packages/wdio-cucumber-framework/src/index.ts

```diff
@@ -35,6 +35,10 @@ import {
 
 import type { SupportCodeLibraryBuilder } from '@cucumber/cucumber/lib/support_code_library_builder/index.js'
 
+export const FILE_PROTOCOL = 'file://'
+
+const cucumberFormatter = url.pathToFileURL(path.resolve(url.fileURLToPath(import.meta.url), '../cucumberFormatter.js')).href
```

Comment:
Feel free to move this into the constructor if this makes it easier for mocking
```suggestion
const cucumberFormatter = url.pathToFileURL(path.resolve(url.fileURLToPath(import.meta.url), '..', 'cucumberFormatter.js')).href
```